contracts/BaseLaunchpeg.sol

```diff
@@ -132,6 +132,22 @@ abstract contract BaseLaunchpeg is
         uint32 _callbackGasLimit
     );
 
+    /// @dev Emitted on setAllowlistStartTime()
+    /// @param oldAllowlistStartTime old allowlist start time
+    /// @param newAllowlistStartTime new allowlist start time
+    event AllowlistStartTimeSet(
+        uint256 oldAllowlistStartTime,
```

Comment:
Why are you putting the old start time in this kind of event ? Is it useful when you're indexing the contract ?

Reply:
oh no, not really. i was following the convention in the JoepegAuctionHouse contract. i don't a preference - i'm fine leaving it out too.

Reply:
As you want, I personally prefer without it but it also makes sense to stay consistent in our contracts

Reply:
ok sure will update to remove the old values